.github/workflows/github-automerge.yaml

```
@@ -0,0 +1,148 @@
+        required: true
+        default: 'main'
+        type: string
+      MERGE_METHOD:
+        description: 'Method to use when merging the pull request into the base branch. Possible values are: merge, rebase, squash.'
+        required: false
+        type: string
+        default: 'squash'
+      MERGE_RETRIES:
+        description: 'Number of times the job will attempt to merge the PR.'
+        required: false
+        type: int
```

actionlint (1.7.10)

[error] invalid value "int" for input type of workflow_call event. it must be one of "boolean", "number", or "string"

(syntax-check)